tempesta_fw/session.h

```diff
@@ -34,6 +34,7 @@
  * session.
  */
 typedef struct {
+	struct sock 	*srv_sk;
```

Comment:
Please comment in the code why this should be outside of ```TfwServer{}``` where it belongs on a first thought.